### PR TITLE
chore: Standardize on the use of "status" vs "quality"

### PR DESCRIPTION
**Pull request by kingthorin**
Related to zaproxy/zaproxy#4732

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: site/content/docs/desktop/start/features/addons.md [hunk 1]
```diff
@@ -22,8 +22,8 @@ Add-ons are assigned a status which may be one of:
 
 |   |         |                                                                                              |
 |---|---------|----------------------------------------------------------------------------------------------|
-|   | Release | which indicates they are of high quality and fit for purpose                                 |
-|   | Beta    | which indicates they are of reasonable quality but may be incomplete or need further testing |
+|   | Release | which indicates they are of high status and fit for purpose                                  |
+|   | Beta    | which indicates they are of reasonable status but may be incomplete or need further testing  |
```
Comment (thc202) [inline site/content/docs/desktop/start/features/addons.md:26]: The spaces should be removed and the vertical line aligned.

Comment (kingthorin) [reply site/content/docs/desktop/start/features/addons.md:26]: They were added to keep it aligned.
quality = 7
status = 6

Comment (kingthorin) [reply site/content/docs/desktop/start/features/addons.md:26]: I guess I could drop one from the alpha line...

Comment (thc202) [reply site/content/docs/desktop/start/features/addons.md:26]: The tables are generated with the minimum length.

File: site/data/addons.yaml [hunk 2]
```diff
@@ -42,7 +42,7 @@
   version: 3
 - id: ascanrules
   name: Active scanner rules
-  description: The release quality Active Scanner rules
```
Comment (psiinon) [inline site/data/addons.yaml:45]: Have these been changed in the actual extensions? If not they will be overwritten when this data is regenerated...

Comment (kingthorin) [reply site/data/addons.yaml:45]: Yup that was a previous PR.

**Review by psiinon — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**